lib/web_console/whiny_object.rb

```diff
@@ -0,0 +1,23 @@
+module WebConsole
+  class WhinyObject < SimpleDelegator
+    def initialize(object, env)
+      @env = env
+      super(object)
+    end
+
+    private
+
+      attr_accessor :env
+
+      def whine_unless(condition)
```

Comment:
I think `log_unless` was a bit clearer. Changing method names because you find them clearer is kinda subjective and just contributes to the diff noise. Think of it as a cosmetic change. Those aren't usually accepted across the rails core projects.